app/src/components/main/analytics/events/common/testItemPages/actionEventsCreators.js

```diff
@@ -78,6 +78,17 @@ export const getClickActionsButtonEvent = (category) => ({
   ...getBasicClickEventParameters(category),
   element_name: 'button_actions',
 });
+export const getClickPencilIconEvent = (category) => ({
+  ...getBasicClickEventParameters(category),
+  icon_name: 'edit_item',
+});
+export const getRefineParametersEvent = (category) => (type, productBugGroup) => ({
```

Comment:
May be rename it to `getRefineParametersEventCreator`?
As this function creates not event, but event creator.
What do you think?

Reply:
I agree!